rmgpy/molecule/moleculeTest.py

```diff
@@ -478,7 +478,7 @@ def testApplyActionDecrementBond(self):
             try:
                 bond.applyAction(action)
             except ActionError:
-                self.assertTrue(order0 < 1)
```

Comment:
So on one of my other branches I ran into a unit test issue with this check, which I fixed by changing 1 to 1.0001, but I guess if this is passing unit tests this is fine.